### PR TITLE
refactor(profiles): simplify check of fully loaded user data

### PR DESCRIPTION
**Pull request by lex111**
The last final part :smile: 

Based on #414 (it too was closed too early, so I had to create a new PR)

### CODE REVIEW

File: src/components/user-profile.component.js [hunk 1]
```diff
@@ -104,99 +107,103 @@ export const UserProfile = ({
...
-          {translate('common.repositories', language)}
-        </Text>
-      </TouchableOpacity>}
-
-      {type !== 'org' && !isNaN(parseInt(starCount, 10)) &&
-        <TouchableOpacity style={styles.unit}>
-          <Text style={styles.unitNumber}>
-            {abbreviateNumber(starCount)}
+    {((user.hasOwnProperty('public_repos') &&
+      !isNaN(parseInt(starCount, 10))) ||
```
Comment (housseindjirdeh) [inline src/components/user-profile.component.js]: Yeah we really want to be using lodash (or a simple utility library) to make this simpler. Please remind me to bring this up after our release

File: src/components/user-profile.component.js [hunk 2]
```diff
@@ -104,99 +107,103 @@ export const UserProfile = ({
...
+              style={styles.unit}
+              onPress={() =>
+                navigation.navigate('FollowingList', {
+                  title: translate('user.following.title', language),
+                  user,
+                  followingCount: user.following > 15 ? 15 : user.following,
+                })}
+            >
+              <Text style={styles.unitNumber}>
+                {!isNaN(parseInt(user.following, 10)) ? user.following : ' '}
```
Comment (housseindjirdeh) [inline src/components/user-profile.component.js]: Similarly here

**Comment by lex111** [comment]
Aah, again, I wanted to make a change :(
However, of course there is my fault, I did not set the WIP label, I'm sorry.

**Comment by lex111** [comment]
Can do a revert?

**Comment by housseindjirdeh** [comment]
No worries, please be careful and try to add WIP labels if you're still working on your PR. Theres' no other way for me to be able to tell :)

Feel free to submit another PR

**Comment by lex111** [comment]
Well, let's then later, still we need to deal with the refactoring of this component, we can better create an issue, so as not to forget about it :smirk:

**Comment by housseindjirdeh** [comment]
Perfect that sounds good to me :D

**Pull request merged**